kbase-extension/scss/partials/_cellToolbar.scss

```diff
@@ -0,0 +1,132 @@
+$default-icon-size: 56px;
+
+.kb-cell-toolbar {
```

Comment:
Moved a ton of css out of the JS file and into its rightful home in a SCSS file.